### ISSUE TITLE
Fix server-side error when attempting to process upload form data

### ISSUE THREAD

**Issue by samuel-thompsonn** [jpiland16/hmv_test#12]
(Note: I mainly added this issue as a convenient way to document progress on this situation. Sorry for any spam emails!)
When you go to the website and upload a file with one or more well-formatted sensors defined, the client gets navigated to the appropriate link, but the file viewer says: "An error occured while loading the data file: The requested file doesn't exist or is of an invalid format. Try reselecting the file or resubmitting." 
I can replicate this by submitting S1-Drill.dat from the Opportunity dataset with time column 0, setting a sensor for the RUA at column 59.

Using pm2 on the server, we can see the following output when a user submits a file:
```
│                                         ││ server > handleUploadedFile threw an error!                                                      │
│                                         ││ server > { Error: write EPIPE                                                                    │
│                                         ││ server >     at afterWriteDispatched (internal/stream_base_commons.js:78:25)                     │
│                                         ││ server >     at writeGeneric (internal/stream_base_commons.js:73:3)                              │
│                                         ││ server >     at Socket._writeGeneric (net.js:714:5)                                              │
│                                         ││ server >     at Socket._write (net.js:726:8)                                                     │
│                                         ││ server >     at doWrite (_stream_writable.js:415:12)                                             │
│                                         ││ server >     at writeOrBuffer (_stream_writable.js:399:5)                                        │
│                                         ││ server >     at Socket.Writable.write (_stream_writable.js:299:11)                               │
│                                         ││ server >     at handleUploadedFile (/home/webs/hmv_test/src/server_side/FormFileProcessor.js:99  │
│                                         ││ server >     at EventEmitter.reader.addEventListener (/home/webs/hmv_test/src/server_side/FormF  │
│                                         ││ server >     at EventEmitter.emit (events.js:203:15) errno: 'EPIPE', code: 'EPIPE', syscall:     │
│                                         ││ server > { Error: spawn python ENOENT                                                            │
│                                         ││ server >     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)           │
│                                         ││ server >     at onErrorNT (internal/child_process.js:415:16)                                     │
│                                         ││ server >     at process._tickCallback (internal/process/next_tick.js:63:19)                      │
│                                         ││ server >   errno: 'ENOENT',                                                                      │
│                                         ││ server >   code: 'ENOENT',                                                                       │
│                                         ││ server >   syscall: 'spawn python',                                                              │
│                                         ││ server >   path: 'python',                                                                       │
│                                         ││ server >   spawnargs:                                                                            │
│                                         ││ server >    [ 'src/server_side/python_programs/multi_sensor_fuser_obj.py',                       │
│                                         ││ server >      '{"sensor_columns":[{"start_column":"59","data_type":"Quaternion"}],"time_column"
```

**Comment by samuel-thompsonn** [comment]
Update: I'm using the `python` command to spawn a Python program using the JS `child_process` API, but I just checked on the server and found that there is no such command:
```
stt13@vcm-20389:/home/webs/hmv_test$ python --version

Command 'python' not found, did you mean:

  command 'python3' from deb python3
  command 'python' from deb python-is-python3

stt13@vcm-20389:/home/webs/hmv_test$ python3 --version
Python 3.8.10
```

So I should probably run using `python3` instead of `python`. However, when I use `python3` manually to run `multi_sensor_fuser_obj.py`, I get an error: `No module named numpy`. I will install numpy and any other dependencies using the VM's package manager or using `pip`.

**Comment by samuel-thompsonn** [comment]
There was an error in the import statement for `AGHandler` in `multi_sensor_fuser_obj.py` where the statement was referring to the wrong package. I am going to put a quick patch directly onto the master branch to fix this.
NOTE: There is also an error where the `multi_sensor_fuser_obj.py` script is splitting by `' '` (a single space) instead of by whitespace. In S1-Drill this leads to columns 59-63 containing an empty string as one of the numbers, which the program presently cannot handle. This fix is coming in a future commit.

**Comment by samuel-thompsonn** [comment]
Right now the server is reporting that it can't run the python script because numpy is not installed. However, I already used a second terminal on the same VM to install pip and numpy, which allows me to run the target python script through the command line. It is not clear why the spawned child process would not have access to the newly installed library.

**Comment by samuel-thompsonn** [comment]
Solved. The issue with numpy was that it was installed in my use directory instead of the directory that the root was using when it ran the server. The simple fix was to use `pip` to uninstall numpy, then run `sudo install numpy` to install it to the correct directory.

I've successfully uploaded S1-Drill.dat (after one additional bug fix for handling exotic whitespace in data files) and it was perfectly viewable--I've since deleted the uploaded file since we already have S1-Drill up.
